railties/lib/rails/generators/rails/app/templates/config/initializers/hsts_subdomains.rb

```diff
@@ -0,0 +1 @@
+Rails.application.config.action_dispatch.hsts_subdomains = true
```

Comment:
Instead of a new option can we set `Rails.application.config.action_dispatch.ssl_options = { subdomains: true }`?